app/javascript/dashboard/assets/scss/widgets/_conversation-view.scss

```diff
@@ -236,7 +236,7 @@
 
   .wrap {
     @include margin($zero $space-normal);
-    max-width: 69%;
+    max-width: 85%;
```

Comment:
As a refactor we have a max-width set for the immediate children `.bubble` also. Can you remove that from there and bring it here and `max-width: min(50rem, 85%)` ?

Reply:
👆 Please check this if works for all attachment types.